tests/unit/s2n_client_psk_extension_test.c

```diff
@@ -1555,7 +1555,7 @@ int main(int argc, char **argv)
      *#     removing any PSKs which are incompatible with the server's
      *#     indicated cipher suite.
      **/
-    {
+    if (s2n_is_tls13_fully_supported()) {
```

Comment:
Why is this only available with TLS1.3?

Reply:
This check is needed because this line expects TLS 1.3:
https://github.com/aws/s2n-tls/blob/3b1e580ec8da8bc14ec0b9332eb88046a7c3ef70/tests/unit/s2n_client_psk_extension_test.c#L1616

And if TLS 1.3 is not fully supported, `s2n_client_hello_send` will downgrade the connection to TLS 1.2.